src/seedu/addressbook/parser/Parser.java

```diff
@@ -17,8 +17,9 @@
 import seedu.addressbook.commands.DeleteCommand;
 import seedu.addressbook.commands.ExitCommand;
 import seedu.addressbook.commands.FindCommand;
+import seedu.addressbook.commands.SortCommand;
```

Comment:
The import classes should also follow the alphabetical order (which means SortCommand should be after ListCommand). For AB Lvl 3 & 4 there will be checkstyles in the travis test.